File: app/src/main/java/br/com/hussan/githubapi/api/ApiInterface.java
package br.com.hussan.githubapi.api;

import java.util.List;

import br.com.hussan.githubapi.models.Repository;
import retrofit2.http.GET;
import retrofit2.http.Path;
import rx.Observable;

/**
 * Created by hussan on 4/11/16.
 */
public interface ApiInterface {

    @GET("/users/{user}/repos")
    Observable<List<Repository>> repositories(
            @Path("user") String user
    );
}

File: app/src/main/kotlin/br/com/hussan/githubapi/data/api/ApiInterface.kt
package br.com.hussan.githubapi.data.api

import br.com.hussan.githubapi.data.model.Repository
import retrofit2.http.GET
import retrofit2.http.Path
import rx.Observable

/**
 * Created by hussan on 4/11/16.
 */
interface ApiInterface {

    @GET("/users/{user}/repos")
    fun repositories(
            @Path("user") user: String): Observable<List<Repository>>
}